src/components/Home.tsx

```diff
@@ -21,22 +21,25 @@ const Title = styled.h1<HeadingProps>`
   ${typography}
 `;
 
-const LangButton = styled.button<HeadingProps>`
+const LangButton = styled.button`
   ${color}
   ${space}
   ${typography}
   ${border}
 `;
 
 const Home = () => {
-  const { t } = useTranslation();
+  const onLangClick = useCallback(
+    (countryId: string) => () => i18next.changeLanguage(countryId),
+    []
+  );
   return (
     <main>
       <Title color="primary" bg="error" p={3} fontSize={3}>
-        {t("home.header")}
+        {onLangClick("home.header")}
```

Comment:
we didnt want to change this one, this was to do with showing the correct text. Which is separate to Buttons that change the active language